sdk/prefix_pool/prefixpool.go

```diff
@@ -6,12 +6,18 @@ import (
 	"sort"
 	"sync"
 
+	"github.com/fsnotify/fsnotify"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
 
 	"github.com/networkservicemesh/networkservicemesh/controlplane/api/connectioncontext"
 )
 
+const (
+	PrefixesFilePath = "/var/lib/networkservicemesh/config/excluded_prefixes.yaml"
```

Comment:
Please use "exclude_prefixes.yaml" constant from previous comment to be specified in one place.

Reply:
- [ ] use "exclude_prefixes.yaml" constant